engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-errata-modal.html

```diff
@@ -27,7 +27,21 @@ <h4 data-block="modal-header">Content Host Errata Management</h4>
       </div>
     </div>
 
-    <div data-extend-template="common/views/katello-agent-notice.html"></div>
+    <div ng-show="katelloAgentPresent" data-extend-template="common/views/katello-agent-notice.html"></div>
+
+    <div bst-feature-flag="remote_actions">
+      <p bst-alert="info" ng-hide="katelloAgentPresent || !hostToolingEnabled">
+        <span translate>
+          If you are not using Remote Execution to manage content on this host, ensure that it has the katello-agent package installed.
```

Comment:
I kinda want to remove this particular message altogether because it is effectively pushing someone to use the agent. Watcha think?

Reply:
I am for removing it, I will take it out and push up the changes.

Reply:
@jturel removed and updated